[2022-09-17 06:06:28,883] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yellow_taxi_data_v2.local_to_gcs_task scheduled__2020-04-02T06:00:00+00:00 [queued]>
[2022-09-17 06:06:28,894] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yellow_taxi_data_v2.local_to_gcs_task scheduled__2020-04-02T06:00:00+00:00 [queued]>
[2022-09-17 06:06:28,895] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-09-17 06:06:28,895] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-09-17 06:06:28,895] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-09-17 06:06:28,907] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2020-04-02 06:00:00+00:00
[2022-09-17 06:06:28,914] {standard_task_runner.py:52} INFO - Started process 5203 to run task
[2022-09-17 06:06:28,920] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'yellow_taxi_data_v2', 'local_to_gcs_task', 'scheduled__2020-04-02T06:00:00+00:00', '--job-id', '50', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_gcs_dag.py', '--cfg-path', '/tmp/tmp67qk6utk', '--error-file', '/tmp/tmpzy7jshk4']
[2022-09-17 06:06:28,921] {standard_task_runner.py:77} INFO - Job 50: Subtask local_to_gcs_task
[2022-09-17 06:06:28,996] {logging_mixin.py:109} INFO - Running <TaskInstance: yellow_taxi_data_v2.local_to_gcs_task scheduled__2020-04-02T06:00:00+00:00 [running]> on host 10d063af5ba7
[2022-09-17 06:06:29,049] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-09-17 06:06:29,076] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=yellow_taxi_data_v2
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2020-04-02T06:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-04-02T06:00:00+00:00
[2022-09-17 06:06:30,607] {python.py:175} INFO - Done. Returned value was: None
[2022-09-17 06:06:30,622] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=yellow_taxi_data_v2, task_id=local_to_gcs_task, execution_date=20200402T060000, start_date=20220917T060628, end_date=20220917T060630
[2022-09-17 06:06:30,659] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-09-17 06:06:30,703] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-09-20 15:34:17,159] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yellow_taxi_data_v2.local_to_gcs_task scheduled__2020-04-02T06:00:00+00:00 [queued]>
[2022-09-20 15:34:17,177] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yellow_taxi_data_v2.local_to_gcs_task scheduled__2020-04-02T06:00:00+00:00 [queued]>
[2022-09-20 15:34:17,177] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-09-20 15:34:17,178] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-09-20 15:34:17,178] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-09-20 15:34:17,197] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2020-04-02 06:00:00+00:00
[2022-09-20 15:34:17,205] {standard_task_runner.py:52} INFO - Started process 793 to run task
[2022-09-20 15:34:17,216] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'yellow_taxi_data_v2', 'local_to_gcs_task', 'scheduled__2020-04-02T06:00:00+00:00', '--job-id', '130', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_gcs_dag.py', '--cfg-path', '/tmp/tmpqvh9c9k_', '--error-file', '/tmp/tmp_6zfe0ut']
[2022-09-20 15:34:17,218] {standard_task_runner.py:77} INFO - Job 130: Subtask local_to_gcs_task
[2022-09-20 15:34:17,304] {logging_mixin.py:109} INFO - Running <TaskInstance: yellow_taxi_data_v2.local_to_gcs_task scheduled__2020-04-02T06:00:00+00:00 [running]> on host 788dccad41b9
[2022-09-20 15:34:17,354] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-09-20 15:34:17,386] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=yellow_taxi_data_v2
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2020-04-02T06:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-04-02T06:00:00+00:00
[2022-09-20 15:34:17,694] {python.py:175} INFO - Done. Returned value was: None
[2022-09-20 15:34:17,727] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=yellow_taxi_data_v2, task_id=local_to_gcs_task, execution_date=20200402T060000, start_date=20220920T153417, end_date=20220920T153417
[2022-09-20 15:34:17,771] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-09-20 15:34:17,844] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-09-21 01:35:28,279] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yellow_taxi_data_v2.local_to_gcs_task scheduled__2020-04-02T06:00:00+00:00 [queued]>
[2022-09-21 01:35:28,295] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yellow_taxi_data_v2.local_to_gcs_task scheduled__2020-04-02T06:00:00+00:00 [queued]>
[2022-09-21 01:35:28,295] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-09-21 01:35:28,295] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-09-21 01:35:28,296] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-09-21 01:35:28,311] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2020-04-02 06:00:00+00:00
[2022-09-21 01:35:28,320] {standard_task_runner.py:52} INFO - Started process 27953 to run task
[2022-09-21 01:35:28,328] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'yellow_taxi_data_v2', 'local_to_gcs_task', 'scheduled__2020-04-02T06:00:00+00:00', '--job-id', '306', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_gcs_dag.py', '--cfg-path', '/tmp/tmpmj5j1k3f', '--error-file', '/tmp/tmp2ccp_gcm']
[2022-09-21 01:35:28,329] {standard_task_runner.py:77} INFO - Job 306: Subtask local_to_gcs_task
[2022-09-21 01:35:28,429] {logging_mixin.py:109} INFO - Running <TaskInstance: yellow_taxi_data_v2.local_to_gcs_task scheduled__2020-04-02T06:00:00+00:00 [running]> on host 788dccad41b9
[2022-09-21 01:35:28,504] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-09-21 01:35:28,533] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=yellow_taxi_data_v2
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2020-04-02T06:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-04-02T06:00:00+00:00
[2022-09-21 01:35:28,835] {python.py:175} INFO - Done. Returned value was: None
[2022-09-21 01:35:28,854] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=yellow_taxi_data_v2, task_id=local_to_gcs_task, execution_date=20200402T060000, start_date=20220921T013528, end_date=20220921T013528
[2022-09-21 01:35:28,911] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-09-21 01:35:28,974] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
